### PR TITLE
Improve instructions of how to run p-scripts

### PR DESCRIPTION
**Pull request by stevepiercy**
- clarify two supported ways to run p-scripts
- add link in each script usage to the instructions. See https://github.com/Microsoft/vscode-python/issues/2427#issuecomment-416330081
- Use explicit lexer for syntax highlighting.
- Add name of script to each heading

### CODE REVIEW

File: docs/narr/commandline.rst [hunk 1]
```diff
@@ -6,15 +6,72 @@ Command-Line Pyramid
...
+
+.. _running-pscripts:
+
+Running p-scripts
+-----------------
+
+All of the Pyramid console scripts may be run either:
+
+*   by its name
+*   through a Python interpreter
```
Comment (mmerickel) [inline docs/narr/commandline.rst]: "as a python module"

Comment (stevepiercy) [reply docs/narr/commandline.rst]: Thanks. Fixed in d7dfe77.

File: docs/narr/commandline.rst [hunk 2]
```diff
@@ -6,15 +6,72 @@ Command-Line Pyramid
...
+Running by ``p*`` script name
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Each of Pyramid's console scripts may be run by its name.  For example:
+
+.. code-block:: bash
+
+    $VENV/bin/pserve development.ini --reload
+
+.. note:: ``$VENV/bin/`` represents the ``bin`` directory in a virtual environment, where ``$VENV`` is an environment variable representing its path.
```
Comment (brettcannon) [inline docs/narr/commandline.rst]: FYI `$VIRTUAL_ENV` is actually set as an environment variable when you are in an activated shell, so if you change it to that then `$VIRTUAL_ENV/bin/pserve` would be copy-and-pastable in UNIX.

**Comment by brettcannon** [comment]
Thanks, @stevepiercy !

File: docs/narr/commandline.rst (hunk 2 above)
Comment (stevepiercy) [reply docs/narr/commandline.rst]: Thanks for the review, @brettcannon!

I clarified the note. `$VENV` is a convention we use throughout Pyramid documentation.

**Pull request merged**